packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```
@@ -185,10 +224,16 @@ utm:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
     expires: never
     data_sensitivity:
-      - web_activity
+      - interaction
 
   term:
-    description: UTM term
+    description: |
+      This metric is similar to the UTM source above; it is used in the Firefox
```

```suggestion
      This metric is similar to the `utm.source`; it is used in the Firefox
```

Nit: remove `above`, it would be confusing when someone's reading this description in Glean Dictionary or Data Catalog.

---

> ... confusing when someone's reading this description in Glean Dictionary or Data Catalog.

Great point.  👍